### DISCUSSION TITLE
How can I dress all bots in bg in a8 set?

### DISCUSSION THREAD

**Discussion by demm500** [trickerer/Trinity-Bots#548] [Q&A]
They run in clothes for pve on bg, from ulduar and from the icecrown citadel.

**Comment by trickerer** [comment]
Gear rules can't be managed that way currently and are hardcoded (except for disabled items which can be managed through DB table).

Wandering bots use any items available for their level (and several levels below, except for level 80 bots). There are only 2 additional rules for level 80: 1) bots have 20% chance to generate all gear using items of ilevel 245 or greater, 2) bots have 50% chance to generate only resilience gear in main slots (PvP set inventory slots).
